### PR TITLE
InfluxDB 2.7.6

### LINKED ISSUE #5417: strong-password parameter/flag for OSS 2.7.6
Will be available in influxDB OSS 2.7.6, April 12th or 15th.

**PR**: https://github.com/influxdata/influxdb/pull/24857, https://github.com/influxdata/influxdb/pull/24887

See linked issue above. New configurable parameter turns on simple password complexity checks

Command line flag `strong-passwords` (and corresponding configuration setting and environment variable) when set to true enables a simple complexity check for use of diferent character classes and length requirements.

##### Relevant URLs
**Issue**: https://github.com/influxdata/influxdb/issues/24856

InfluxDB OSS 2.7.6 (and subsequent)

### PR DESCRIPTION
**Pull request by sanderson**
Closes #5417

- Adds 2.7.5 and 2.7.6 release notes
- Add strong passwords `influxd` config option

---

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2/reference/release-notes/influxdb.md [hunk 1]
```diff
@@ -8,6 +8,45 @@ menu:
...
+- Update constant time code to make password strength calculations more constant.
+
+### Features
+
+- Add optional [stricter password requirements](/influxdb/v2/reference/config-options/#strong-passwords).
+
+### Maintenance
+
+- Upgrade to Go 1.21.8.
+- Upgrade Go toolchain to 1.21.9.
```
Comment (davidby-influx) [inline content/influxdb/v2/reference/release-notes/influxdb.md:29]: I think we should clarify here that the build is with 1.21.9.  I find the two contradictory statements (while true), confusing.

**Review by davidby-influx — CHANGES_REQUESTED** [review]
One clarification requested.

File: content/influxdb/v2/reference/release-notes/influxdb.md (hunk 1 above)
Comment (sanderson) [inline content/influxdb/v2/reference/release-notes/influxdb.md:29]:
```suggestion
- Upgrade Go toolchain (for building release artifacts) to 1.21.9.
```

**Review by davidby-influx — APPROVED** [review]
Not blocking, but I would just omit the 1.21.8 line. Those changes are replaced by 1.21.9.

File: content/influxdb/v2/reference/config-options.md [hunk 2]
```diff
@@ -2750,6 +2751,58 @@ store = "bolt"
...
+}
+```
+{{% /code-tab-content %}}
+{{< /code-tabs-wrapper >}}
+
+---
+
 ### testing-always-allow-setup
 Ensures the `/api/v2/setup` endpoint always returns `true` to allow onboarding.
 This configuration option is primary used in continuous integration tests.
```
Comment (jstirnaman) [inline content/influxdb/v2/reference/config-options.md:2808]:
```suggestion
This configuration option is primarily used in continuous integration tests.
```

File: content/influxdb/v2/reference/release-notes/influxdb.md [hunk 3]
```diff
@@ -8,6 +8,45 @@ menu:
...
+- Upgrade to Go 1.21.8.
+- Upgrade Go toolchain (for building release artifacts) to 1.21.9.
+
+## v2.7.5 {date="2024-01-05"}
+
+### Bug Fixes
+
+- Only execute `init_config` on install.
+- Enable `Secure` when using TLS and enable `HttpOnly`.
+- Correctly return 4xx errors instead of 5xx errors.
```
Comment (jstirnaman) [inline content/influxdb/v2/reference/release-notes/influxdb.md:37]:
```suggestion
- Correctly return `4xx` errors instead of `5xx` errors.
```

**Review by jstirnaman — APPROVED** [review]
Looks good. A couple of small fixes.

File: content/influxdb/v2/reference/release-notes/influxdb.md [hunk 4]
```diff
@@ -8,6 +8,45 @@ menu:
...
+- Update constant time code to make password strength calculations more constant.
+
+### Features
+
+- Add optional [stricter password requirements](/influxdb/v2/reference/config-options/#strong-passwords).
+
+### Maintenance
+
+- Upgrade to Go 1.21.8.
+- Upgrade Go toolchain (for building release artifacts) to 1.21.9.
```
Comment (sanderson) [inline content/influxdb/v2/reference/release-notes/influxdb.md:29]:
```suggestion
- Upgrade to Go to 1.21.9.
```

**Pull request merged**